dataplane/README.md

```diff
@@ -0,0 +1,155 @@
+# The Lemming Dataplane
+
+## Overview
+
+The dataplane consists of a several components:
+
+1. forwarding: packet processing actions, tables, and ports.
+1. saiserver: implements a "proto-ized" version of the [SAI api](https://github.com/opencomputeproject/SAI) and defines the forwarding pipeline
```

Comment:
mention SAI full name for the first time

Reply:
done